client/client.go

```diff
@@ -104,7 +110,7 @@ func NewClientWithUserAgent(apiKey string, orgName string, env string, userAgent
 		orgName:     orgName,
 		baseURL:     fullBaseURL,
 		userAgent:   userAgent,
-		rateLimiter: rate.NewLimiter(rate.Limit(DefaultRateLimitPerSecond), 1),
+		rateLimiter: rate.NewLimiter(rate.Limit(rateLimit), 1),
```

Comment:
(understanding check): do your changes solely allow higher rate limits in testing (ie, 'real' clients will still have the default of 2 requests/sec)?

Reply:
Not really, I've just made it configurable and increased it when running tests locally.

Reply:
Oh sorry, I phrased that weirdly. I think I just wanted to make sure I understand the current config. Is this right for how rate limits are configured today?
- local: 100 req/s
- acceptance test: 5 req/s
- public (non-testing): 2 req/s

Reply:
Ohh, yeah, that's right!